tests/integration/test_verify.py

```diff
@@ -16,6 +17,24 @@ def test_verify_valid_sources(source_dir):
     assert exit_code == 0
 
 
+@pytest.mark.parametrize('source_dir,output', [
+    ("tests/test_files/yaml_source_dir/valid_yamls_with_multiple_crds",
+     "tests/bundle.yaml"),
+])
+def test_verify_with_output_file(source_dir, output):
+    process = subprocess.Popen(f'operator-courier verify {source_dir} --output={output}',
+                               shell=True,
+                               stdout=subprocess.PIPE,
+                               stderr=subprocess.STDOUT)
+    exit_code = process.wait()
+    assert exit_code == 0
+
+    fileCreated = os.path.isfile(output)
+    assert fileCreated
```

Comment:
Should we update some variable naming styles according to [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)? I've seen mixed use of camelCase and snake_case in our project.

Reply:
This is a great point, pr updated.